specifyweb/businessrules/rules/cojo_rules.py

```diff
@@ -50,4 +58,15 @@ def cog_post_save(cog):
             childcog=cog,
             childco=None,
             parentcog=cog.parentcog
+        )
+
+@orm_signal_handler('post_save', 'Collectionobject')
+def cog_post_save(co):
```

Comment:
Can you rename this function as we already have a `cog_post_save` function in this file? It shouldn't make any difference functionally but it would be less confusing